staged-dependencies.R

```diff
@@ -51,34 +51,74 @@ options(
   staged.dependencies.token_mapping = split_to_map(token_mapping)
 )
 
-
-if (file.exists("renv.lock")) {
-  renv::restore()
-} else {
-  remotes::install_deps(dependencies = TRUE, upgrade = "never", Ncpus = threads)
-}
+# Install and run staged.dependencies if config file exists
 if (file.exists("staged_dependencies.yaml")) {
   cat("Install Staged Dependencies\n\n")
   if (!require("staged.dependencies")) {
```

Comment:
If you have a different version to `sd_version` installed then it won't be sd_version that is used you probably want to check version (or just force the intall)

Reply:
Yeah good point.